install-spark.yaml

```diff
@@ -41,6 +41,7 @@
         - scala
         - git
         - unzip  
+        - build-essential  
```

Comment:
Did you add the equivalent packages for Centos?

Reply:
i didnt, couldnt find the exact match for it at this time

Reply:
The correct package for Centos is "@development-tools"

Reply:
do you want me to add that or are you taking care of that?